arch/arm/core/aarch64/switch.S

```diff
@@ -36,6 +36,10 @@ GDATA(_k_neg_eagain)
  * For Cortex-A, PendSV exception is not supported by the architecture and this
  * function is directly called either by _isr_wrapper() in case of preemption,
  * or z_arm64_svc() in case of cooperative switching.
+ *
+ * With USE_SWITCH we expect:
+ *  - x17 <- old_thread->switch_handle
+ *  - x16 <- new_thread->switch_handle
  */
 GTEXT(z_arm64_pendsv)
 SECTION_FUNC(TEXT, z_arm64_pendsv)
```

Comment:
could we please re-name this stuff, since we are not actually doing the exception dance that aarch32 is doing?

Reply:
Yeah, it's time to rename that stuff. V2 coming.